FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,6 +5,10 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
+	enum Constants {
```

Comment:
`Constants` is an implementation detail of `RemoteFeedLoader`, so it should be marked as `private`:
```suggestion
	private enum Constants {
```